src/pages/gallery/index.module.scss

```diff
@@ -1,5 +1,12 @@
+@import "~bootstrap/scss/_functions.scss";
+@import "~bootstrap/scss/_variables.scss";
+
 .grid {
   display: grid;
   grid-template-columns: repeat(auto-fill, minmax(16rem, 1fr));
   grid-gap: 1rem;
 }
+
+.container {
+  background-color: $gray-200;
+}
```

Comment:
I believe this is unused.

Reply:
This is used, to add gray background to gallery section.